### DISCUSSION TITLE
Allow to move non-food products to Open Beauty Facts, Open Pet Food Facts, Open Products Facts

### DISCUSSION THREAD

**Discussion by teolemon** [openfoodfacts/robotoff#1211] [Ideas]
Create (and apply ?) non-food-products category insight for detected cosmetics and petfood.
That will disable Nutri-Score and NOVA computations
There's an API to do that, works in get
`*/cgi/product_jqm.pl?type=edit&code=7611330103090&new_code=obf`

**Comment by raphael0202** [comment]
Shouldn't we move petfood and cosmetic products to OPFF and OBF respectively?

**Comment by teolemon** [comment]
There's an API to do that, works in get
*/cgi/product_jqm.pl?type=edit&code=7611330103090&new_code=obf
